stacker/providers/aws/default.py

```diff
@@ -895,12 +895,11 @@ def get_stack_tags(self, stack, **kwargs):
     def get_outputs(self, stack_name, *args, **kwargs):
         if stack_name not in self._outputs:
             stack = self.get_stack(stack_name)
-            self.set_outputs(stack_name, stack)
+            self._outputs[stack_name] = get_output_dict(stack)
         return self._outputs[stack_name]
 
-    def set_outputs(self, stack_name, stack):
-        self._outputs[stack_name] = get_output_dict(stack)
-        return
+    def get_output_dict(self, stack):
```

Comment:
Is this used anywhere?

Reply:
Yeah, this gets used in the actions to strip outputs from `provider_stack`. I'd rather `providers` just return something like a `stacker.providers.base.Stack` object instead, but that's a bigger refactor.